client/shaders/nodes_shader/opengl_vertex.glsl

```diff
@@ -262,14 +265,14 @@ void main(void)
 
 		if (f_timeofday < 0.2) {
 			adj_shadow_strength = f_shadow_strength * 0.5 *
-				(1.0 - mtsmoothstep(0.18, 0.2, f_timeofday));
```

Comment:
Can you explain the magic number changes?

Reply:
Simple reason is the old values don't look very good imo, especially during sunsets the shadows fade far too quickly. This is basically a function for shadow strength over time, so I've plotted that here. Blue is the original, red is the changed one. 

![functions](https://github.com/minetest/minetest/assets/72752000/62249932-9587-4e90-95e4-d37423debc70)

It should be pretty apparent why sunsets would be especially washed out. The old function is also very strangely asymmetric, which is odd if you think about the physical implications of that, and the new one isn't. I suppose it makes sense that the shadows change faster when the moon rises and sets compared to the sun, so that might make sense to change back to be fair.